internal/operators/cnv/cnv_operator.go

```diff
@@ -91,8 +91,8 @@ func (o *operator) ValidateHost(ctx context.Context, cluster *common.Cluster, ho
 
 	// If the Role is set to Auto-assign for a host, it is not possible to determine whether the node will end up as a master or worker node.
 	if host.Role == models.HostRoleAutoAssign {
-		status := "All host roles must be assigned to enable CNV."
-		o.log.Info("Validate Requirements status ", status)
+		status := fmt.Sprintf("Host role must be assigned to enable CNV for host %s in cluster %s", host.ID, cluster.ID)
```

Comment:
I don't think it's the correct fix for 2 reasons:
1. after this change the user will see in the UI the validation message with the host id & cluster id and it's important only for the logs - not for the validation message
2. I'm not sure we need this log line for debugging - we can get this info from the DB - we have it in the host.validation_info field or we can see it in the UI

Reply:
Thanks for the explanation. In that case, I"ll remove that line from log.